yb-voyager/cmd/import.go

```diff
@@ -238,7 +240,17 @@ func registerImportDataToTargetFlags(cmd *cobra.Command) {
 If any table on YugabyteDB database is non-empty, it prompts whether you want to continue the import without truncating those tables; 
 If you go ahead without truncating, then yb-voyager starts ingesting the data present in the data files with upsert mode.
 Note that for the cases where a table doesn't have a primary key, this may lead to insertion of duplicate data. To avoid this, exclude the table using the --exclude-file-list or truncate those tables manually before using the start-clean flag (default false)`)
-	BoolVar(cmd.Flags(), &truncateTables, "truncate-tables", false, "Truncate tables on target YugabyteDB before importing data. Only applicable along with --start-clean true (default false)")
+	BoolVar(cmd.Flags(), &truncateTables, "truncate-tables", false,
+		"Truncate tables on target YugabyteDB before importing data. Only applicable along with --start-clean true (default false)")
+
+	// TODO: restrict changing of flag value after import data has started
+	// TODO: Detailed description of the flag
+	cmd.Flags().StringVar(&tconf.OnPrimaryKeyConflictAction, "on-primary-key-conflict", "ERROR",
+		`Action to take on primary key conflict during data import.
+Supported values:
+  ERROR(default): Imports data in this mode fails if any primary key conflict is encountered, assuming such conflicts are unexpected.
+  IGNORE		: Skips rows that already exist with the same primary key and enables fast-path import, which provides significantly better performance for large data loads.`)
```

Comment:
- let's not use the word "significant".
- let's also qualify the exact case: "better performance for colocated tables"

Reply:
done